lm75bd/lm75bd.c

```diff
@@ -26,8 +27,31 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  error_code_t errCode;
+
+  /* Selecting temperature register */
+  uint8_t sendData = 0;
```

Comment:
Define the temperature register (`LM75BD_REG_TEMP`) under LM75BD_REG_CONF at the top of this file. Use that instead of hard-coding 0 here.

Reply:
Done, will push change.